### DISCUSSION TITLE
How do I disable parameter highlighting?

### DISCUSSION THREAD

**Discussion by cactusbento** [LazyVim/LazyVim#4632] [Q&A] [answered]
![LazyVim_highlighting](https://github.com/user-attachments/assets/61ee5ce9-1571-459b-b54d-372489088fba)

Image says all. The highlight appears no matter what, and unless I hit `<tab>` while on the same line, it will just follow the cursor until I `:q`.

**Comment by dotfrag** [comment, marked as answer]
Duplicate https://github.com/LazyVim/LazyVim/discussions/4636.

**Comment by cactusbento** [comment]
For anyone looking, I just enabled the `LuaSnip` extra after reading #4636. That got rid of the behavior.
